components/homekit/esp_hap_core/src/esp_hap_ip_services.c

```diff
@@ -1312,6 +1312,7 @@ static void hap_send_notification(void *arg)
     }
     /* If no characteristic notifications are pending, just exit */
     if (i == 0) {
+	hap_platform_memory_free(char_arr); // Memory leak!
```

Comment:
Can you change the comment here so that it can be merged as is?

Reply:
I deleted this comment because everything is rather understandable.